redbot/cogs/streams/streams.py

```diff
@@ -356,27 +373,31 @@ async def streamalert_stop(self, ctx: commands.Context, _all: bool = False):
     @streamalert.command(name="list")
     async def streamalert_list(self, ctx: commands.Context):
         """List all active stream alerts in this server."""
-        streams_list = defaultdict(list)
+        streams_list = defaultdict(dict)
```

Comment:
(this comment is optional) You could use nested defaultdict for `streams_list` instead and then not have to use `.setdefault()` on it later on:
```suggestion
        streams_list = defaultdict(lambda: defaultdict(list))
```